<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comL"/>
    </item>
    <item>
     <widget class="QPushButton" name="OpenB">
      <property name="text">
       <string>Открытие канала</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CheckB">
      <property name="text">
       <string>Проверка вывода</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="startb">
      <property name="text">
       <string>Старт записи</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="endb">
      <property name="text">
       <string>Конец записи</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
